cpp/src/aztec3/circuits/rollup/base/.test.cpp

```diff
@@ -219,12 +226,29 @@ class base_rollup_tests : public ::testing::Test {
 TEST_F(base_rollup_tests, no_new_contract_leafs)
 {
     // When there are no contract deployments. The contract tree should be inserting 0 leafs, (not empty leafs);
+    // Initially, the start_contract_tree_snapshot is empty (leaf is 0. hash it up).
+    // Get sibling path of index 0 leaf (for circuit to check membership via sibling path)
+    // No contract leaves -> will insert empty tree -> i.e. end_contract_tree_root = start_contract_tree_root
+
     BaseRollupInputs emptyInputs = getEmptyBaseRollupInputs();
-    BaseRollupPublicInputs outputs = aztec3::circuits::rollup::native_base_rollup::base_rollup_circuit(emptyInputs);
+    auto empty_contract_tree = native_base_rollup::MerkleTree(CONTRACT_TREE_HEIGHT);
+    // fetch sibling path from hash path (only get the second half of the hash path)
+    auto hash_path_of_0 = empty_contract_tree.get_hash_path(0);
+    std::array<NT::fr, CONTRACT_SUBTREE_INCLUSION_CHECK_DEPTH> sibling_path_of_0;
+    for (size_t i = 0; i < CONTRACT_SUBTREE_INCLUSION_CHECK_DEPTH; ++i) {
```

Comment:
You can use a fucntion to translate from hash-path to sibling path to make it a bit easier to figure out, and handle non 0 cases.
```
template <size_t N> std::array<fr, N> get_sibling_path(stdlib::merkle_tree::MemoryTree tree, size_t leafIndex)
{
    std::array<fr, N> siblingPath;
    auto path = tree.get_hash_path(leafIndex);

    for (size_t i = 0; i < N; i++) {
        //
        if (leafIndex & (1 << i)) {
            siblingPath[i] = path[i].first;
        } else {
            siblingPath[i] = path[i].second;
        }
    }
    return siblingPath;
}
```

Reply:
I've also added get_frontier_path to memory trees in barretenberg but it is not merged yet, we can move to that branch for the meantime and remove this interrmediate step